src/SourceBuild/content/test/Microsoft.DotNet.Installer.Tests/LinuxInstallerTests.cs

```diff
@@ -0,0 +1,373 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.VisualStudio.TestPlatform.Utilities;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Net.Http;
+using System.Runtime.Intrinsics.Arm;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using TestUtilities;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.Installer.Tests;
+
+public class LinuxInstallerTests : IDisposable
+{
+    private readonly DockerHelper _dockerHelper;
+    private readonly string _tmpDir;
+    private readonly string _contextDir;
+
+    private bool _rpmContextInitialized = false;
+    private bool _debContextInitialized = false;
+    private bool _sharedContextInitialized = false;
+
+    private const string NetStandard21RpmPackage = @"https://dotnetcli.blob.core.windows.net/dotnet/Runtime/3.1.0/netstandard-targeting-pack-2.1.0-x64.rpm";
+    private const string NetStandard21DebPackage = @"https://dotnetcli.blob.core.windows.net/dotnet/Runtime/3.1.0/netstandard-targeting-pack-2.1.0-x64.deb";
+    private const string RuntimeDepsRepo = "mcr.microsoft.com/dotnet/nightly/runtime-deps";
+    private const string RuntimeDepsVersion = "10.0-preview";
+
+    public static bool IncludeRpmTests => Config.TestRpmPackages;
+    public static bool IncludeDebTests => Config.TestDebPackages;
+
+    private enum PackageType
+    {
+        Rpm,
+        Deb
+    }
+
+    private ITestOutputHelper OutputHelper { get; set; }
```

Comment:
Curious why a property is used here but fields for all other values?

Reply:
I did originally use a field, but changed to property to match the borrowed `DockerHelper` class. There is no need for that matching, of course. I'll change it back to field.